torch/csrc/autograd/engine.h

```diff
@@ -272,7 +272,8 @@ struct TORCH_API Engine {
       const variable_list& inputs,
       bool keep_graph,
       bool create_graph,
-      const edge_list& outputs = {});
+      const edge_list& outputs = {},
+      bool accumulate_grad = false);
```

Comment:
nit: not sure you actually want default values for this.
This is an "internal" API so we prefer to just update all the callsites so that we're sure we don't miss any (which i think are missed in this PR).